mobile/android/android-emulator-services.ts

```diff
@@ -184,9 +184,13 @@ class AndroidEmulatorServices implements Mobile.IAndroidEmulatorServices {
 
 		// if we get here, there's at least one running emulator
 		let getNameFunction = this.$options.geny ? this.getNameFromGenymotionEmulatorId : this.getNameFromSDKEmulatorId;
-		let emulatorId = await _(runningEmulators).find(emulator => getNameFunction.apply(this, [emulator]) === image);
-
-		return emulatorId;
+		for (let emulatorId of runningEmulators) {
+			const currentEmulatorName = await getNameFunction.apply(this, [emulatorId]);
+			if (currentEmulatorName === image) {
+				return emulatorId;
+			}
+		}
+		this.$errors.failWithoutHelp("Couldn't find emulator id");
```

Comment:
This changes the behavior of the method. Previously it's been returning undefined. And the call to the method relies on this undefined value:
https://github.com/telerik/mobile-cli-lib/blob/master/mobile/android/android-emulator-services.ts#L269

Is there a particular reason to add this failure here?